README.md

```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
 
 [Feature Requests](http://feathub.com/jellyfin/jellyfin)
 
-## Building Jellyfin packages
+## Prebuilt Jellyfin packages
+
+Prebuild packages are available for Debian/Ubuntu and Arch, and via Docker Hub.
+
+### Docker
+
+The Jellyfin Docker image is available on Docker Hub at https://hub.docker.com/r/jellyfin/jellyfin/
+
+### Arch
+
+The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfin-git/
+
+### Debian/Ubuntu
+
+A package repository is available at https://repo.jellyfin.org. To use it:
+
+0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Import the GPG signing key (signed by Joshua):
+    ```
+    wget -O - https://repo.jellyfin.org/debian/jellyfin-signing-key-joshua.gpg.key | sudo apt-key add -
+    ```
+0. Add an entry to `/etc/sources.list.d/jellyfin.list`:
```

Comment:
At least on Ubuntu, I believe this should be `/etc/apt/sources.list.d/jellyfin.list`

Reply:
It is for Debian too - typo on my part. Fixed in an upcoming commit.